### ISSUE TITLE
Doctest tests are not skipped correctly

### ISSUE THREAD

**Issue by bilderbuchi** [matepek/vscode-catch2-test-adapter#309]
A test that is marked with `doctest::skip()`, but when I "Run all the tests", this test is also run when it should be skipped.

The problem is that doctest does not tell us if a test is marked as skipped at enumeration stage: 
`./doctest1.exe --reporters=xml --list-test-cases --no-skip=true`
We need an xml attribute for the TestCase in case it's skipped.

There is an upstream issue about this at https://github.com/doctest/doctest/issues/320 -- when that is solved, we can solve this on our side.

_Originally posted by @matepek in https://github.com/matepek/vscode-catch2-test-adapter/issues/283#issuecomment-930352846_

**Comment by bilderbuchi** [comment]
With the latest doctest and this extension's release, tests marked as such are skipped! 😄 

Curiously, the labeling in the testing view is only applied when the tests are actually _run_ (and consequently skipped), no amount of "refreshing the tests" or restarting VSCode helped. I suspect that this one I'd have to lay at the feet of VSCode, though.

**Comment by matepek** [comment]
"labeling" you mean "labeling skipped tests"?
Could be solved but I'm blaming the vscode for the api design 🙃

**Comment by bilderbuchi** [comment]
Yeah, indeed, with this arced arrow symbol. No big deal, there's bigger fish to fry there.
Thanks for your work!

**Comment by matepek** [comment]
enjoy
